src/test/skript/tests/syntaxes/effects/EffHealth.sk

```diff
@@ -21,3 +21,9 @@ test "health effect":
 	heal {_m} by 99
 	assert health of {_m} is 5 with "heal cow ##3 failed"
 	clear all entities
+
+	set {_item1} and {_item1Copy} to diamond sword with damage 100
+	set {_item2} and {_item2Copy} to iron sword
+	repair {_item2} and {_item1}
+	assert {_item1} is iron sword to fail with "{_item1} is an iron sword even though it was diamond before repair"
+	assert {_item2} is {_item2Copy} with "{_item2} was no longer the same item"
```

Comment:
This part should really be a regression test for this particular issue (the regression tests help us to track historically-problematic cases -- and why something is the way it is -- when something changes in future)